codegen/src/api/calls.rs

```diff
@@ -35,13 +34,51 @@ use quote::{
 };
 use scale_info::form::PortableForm;
 
+/// Generate calls from the provided pallet metadata.
+///
+/// The function creates a new module named `calls` under the pallet's module.
+/// ```ignore
+/// pub mod PalletName {
+///     pub mod calls {
+///     ...
+///     }
+/// }
+/// ```
+///
+/// The function generates the calls as rust structs that implement the `subxt::Call` trait
+/// to uniquely identify the call's identity when creating the extrinsic.
+///
+/// ```ignore
+/// pub struct CallName {
+///      pub call_param: type,
+/// }
+/// impl ::subxt::Call for CallName {
+/// ...
+/// }
+/// ```
+///
+/// Calls are extracted from the API as wrapped into the generated `TransactionApi` of
```

Comment:
```suggestion
/// Calls are extracted from the API and wrapped into the generated `TransactionApi` of
```

Does this make more sense? Not sure if this is the right verbage.